src/Tickets/Seating/app/admin/maps/actions.js

```diff
@@ -0,0 +1,48 @@
+import {ajaxUrl, ajaxNonce} from "../../service/service-api/externals";
+import {onReady, getLocalizedString} from "../../utils";
+
+const getString = (key) => getLocalizedString(key, 'maps');
+
+const register_delete_action = () => {
```

Comment:
Functions should be declared as `function`, not constants.

For the purpose of testing, they should be exported.

Reply:
Functions should also have comments following the documentation standard and JSDoc.